core/src/main/java/com/orange/ouds/core/theme/OudsColorScheme.kt

```diff
@@ -0,0 +1,1121 @@
+    return when (token) {
+        OudsColorKeyToken.Decorative.Accent1Default -> decorativeColors.accent1Default
+        OudsColorKeyToken.Decorative.Accent1Emphasized -> decorativeColors.accent1Emphasized
+        OudsColorKeyToken.Decorative.Accent1Muted -> decorativeColors.accent1Muted
+        OudsColorKeyToken.Decorative.Accent2Default -> decorativeColors.accent2Default
+        OudsColorKeyToken.Decorative.Accent2Emphasized -> decorativeColors.accent2Emphasized
+        OudsColorKeyToken.Decorative.Accent2Muted -> decorativeColors.accent2Muted
+        OudsColorKeyToken.Decorative.Accent3Default -> decorativeColors.accent3Default
+        OudsColorKeyToken.Decorative.Accent3Emphasized -> decorativeColors.accent3Emphasized
+        OudsColorKeyToken.Decorative.Accent3Muted -> decorativeColors.accent3Muted
+        OudsColorKeyToken.Decorative.Accent4Default -> decorativeColors.accent4Default
+        OudsColorKeyToken.Decorative.Accent4Emphasized -> decorativeColors.accent4Emphasized
+        OudsColorKeyToken.Decorative.Accent4Muted -> decorativeColors.accent4Muted
+        OudsColorKeyToken.Decorative.Accent5Default -> decorativeColors.accent5Default
+        OudsColorKeyToken.Decorative.Accent5Emphasized -> decorativeColors.accent5Emphasized
+        OudsColorKeyToken.Decorative.Accent5Muted -> decorativeColors.accent5Muted
+        OudsColorKeyToken.Decorative.BrandPrimary -> decorativeColors.brandPrimary
+        OudsColorKeyToken.Decorative.BrandSecondary -> decorativeColors.brandSecondary
+        OudsColorKeyToken.Decorative.BrandTertiary -> decorativeColors.brandTertiary
+        OudsColorKeyToken.Decorative.NeutralDefault -> decorativeColors.neutralDefault
+        OudsColorKeyToken.Decorative.NeutralEmphasized -> decorativeColors.neutralEmphasized
+        OudsColorKeyToken.Decorative.NeutralMuted -> decorativeColors.neutralMuted
+        OudsColorKeyToken.Decorative.SkinTint100 -> decorativeColors.skinTint100
+        OudsColorKeyToken.Decorative.SkinTint200 -> decorativeColors.skinTint200
+        OudsColorKeyToken.Decorative.SkinTint300 -> decorativeColors.skinTint300
+        OudsColorKeyToken.Decorative.SkinTint400 -> decorativeColors.skinTint400
+        OudsColorKeyToken.Decorative.SkinTint500 -> decorativeColors.skinTint500
+        OudsColorKeyToken.Decorative.SkinTint600 -> decorativeColors.skinTint600
+        OudsColorKeyToken.Decorative.SkinTint700 -> decorativeColors.skinTint700
+        OudsColorKeyToken.Decorative.SkinTint800 -> decorativeColors.skinTint800
+        OudsColorKeyToken.Decorative.SkinTint900 -> decorativeColors.skinTint900
+    }
+}
+
+@Stable
+fun OudsColorScheme.fromToken(token: OudsColorKeyToken.Global): Color {
+    return when (token) {
+        OudsColorKeyToken.Global.TransparentDefault -> globalColors.transparentDefault
+    }
+}
+
+val OudsColorSemanticTokens.materialLightColorScheme: ColorScheme
+    get() = lightColorScheme(
```

Comment:
Are these final mapping from Maxime? Or is this temporary?

Reply:
This is temporary. We are waiting for him to fix these values.